### DISCUSSION TITLE
Inquiry on Notions compatibility

### DISCUSSION THREAD

**Discussion by greastern** [tim-hub/obsidian-bible-reference#174] [Ideas]
I've been really blessed with this plug-in in obsidian and I mainly use it for creating sermon notes, devotions, and general studying of scripture. 

I'm noticing that notion is a bit easier when it comes to collaborating with others in a interactive study method. 

Is there a way to bring the same sort of capability of calling in bible verses into notion? Even the way it's displayed in obsidian makes it easy to differentiate notes from scriptures!

**Comment by tim-hub** [comment]
Thanks for the feedback.

It would be a good idea. I did not investigate too much in Notion plugin system, current scope of this project is to focus on Obsidian
